src/components/highlighted-places/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { useHighlightedPlaces, useLocation } from 'containers/datasets/locations/hooks';
+
+const HIGHLIGHTED_PLACES_IMAGES = {
+  '0edd0ebb-892b-5774-8ce5-08e0ba7136b1': 'rufiji.jpg', //: RufijiImage,
+  '4a79230b-7ecb-58ae-ba0d-0f57faa2a104': 'saloum.png', //: SaloumImage,
+};
+
+const HighlightedPlaces = () => {
+  const { data } = useHighlightedPlaces();
+  const {
+    query: { locationType, id },
+  } = useRouter();
+  const {
+    data: { location_id },
+  } = useLocation(locationType, id);
+  const isHighlightedPlace = !!HIGHLIGHTED_PLACES_IMAGES[location_id];
+
+  return (
+    <div className="flex justify-between space-x-6">
+      {data.map((d) => {
+        if (location_id !== d.location_id)
+          return (
+            <Link
+              key={d.location_id}
+              href={`/${d.location_type}/${d.location_id}`}
+              className="flex flex-1"
+            >
+              <div
+                key={d.location_id}
+                className={`flex h-60 flex-1 rounded-[20px] bg-[url('/images/highlighted-places/${
```

Comment:
`bg-[url('/images/highlighted-places/${whatever}`does not work. Check https://tailwindcss.com/docs/background-image#customizing-your-theme